contracts/FeralfileArtworkV4_2.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import { FeralfileExhibitionV4_1 } from "./FeralfileArtworkV4_1.sol";
+import { Base64 } from "@openzeppelin/contracts/utils/Base64.sol";
+
+contract FeralfileExhibitionV4_2 is FeralfileExhibitionV4_1 {
+
+    error TokenIDNotFound();
+    error MintNotEnabled();
+
+    struct TokenInfo {
+        string thumbnail;
```

Comment:
Same [here](https://github.com/bitmark-inc/feralfile-exhibition-smart-contract/pull/41/files#r1611369888), could it be `imageURI`?